Tw/moderator guide

shereefb opened the pull request:
Game over conditions and very rough moderator guide

@tannerwelsh please review and merge

shereefb committed:
no collusion

shereefb committed:
adds more moderation notes from retreat

shereefb committed:
scratch notes from tanner's moderation

shereefb committed:
adds more scratch notes to moderator guide

shereefb committed:
adds more scratch notes for first week moderator guide

shereefb committed:
adds notes from second week

shereefb committed:
adding game boundaries

shereefb committed:
small edits to boundaries

shereefb committed:
some random notes

shereefb committed:
merges master

shereefb committed:
updates summary

tannerwelsh committed:
Language and formatting changes

Assessment Accuracy -> Discernment
Avg. Hours per Week -> Time Dedication per Cycle

Updated the number type for support scores, and made other type and
formatting edits.

tannerwelsh committed:
Clarify fail conditions (renamed from Game Over)

Also change filename (no need for duplicate "Game")

tannerwelsh committed:
Fix broken link in FAQ

tannerwelsh committed:
Move stats to own file; summarize boundaries; add success conditions

tannerwelsh committed:
Reference Stats and Boundaries from README

tannerwelsh commented on Game_Manual/Boundaries.md:
```diff
@@ -0,0 +1,43 @@
+
+## Success Conditions
+
+A player succeeds when they (a) meet the above objectives and (b) fulfill the obligations in their contract.
```
Is this accurate / clear?

tannerwelsh commented on Game_Manual/Boundaries.md:
```diff
@@ -0,0 +1,43 @@
+1. [Culture Contribution][culture-contribution] is too low
+1. [Discernment][discernment] is too high or too low
+
+If these conditions are met, the Moderator will expel the player from the game.
```
I can't figure out what word is best for this. I don't like "kick out" (kinda physically violent) or "expel" (too school-related), but I prefer expel for now.

tannerwelsh committed:
Minor clarification

tannerwelsh commented:
@shereefb I made some non-trivial changes that I want your eyes on before merging. If you're ok with them, feel free to merge.

Highlights:
- rename Assessment Accuracy -> Discernment
- rename Average Hours per Week -> Time Dedication per Cycle (IMHO we shouldn't be looking at average hours, but consistently low hours in multiple cycles)
- changed stat representation for Learning Support and Culture Contribution (percent 0..100% not 1..7, see comment https://github.com/LearnersGuild/game/issues/111#issuecomment-236168664 for rationale)
- made the fail conditions trigger when one "persists over multiple cycles"

Commit:
Update Boundaries.md

Commit:
changes td per cycle to per week

shereefb commented:
@tannerwelsh changes look good. I really like them. 

Switched TD per cycle, to TD per week. And removed it from fail conditions! I think it's rolled up in XP and we don't need it. We can always add it later.

Commit:
Removes time dedication per week as fail condition
